src/hotspot/share/runtime/objectMonitor.cpp

```diff
@@ -545,7 +545,7 @@ bool ObjectMonitor::deflate_monitor() {
     // Java threads. The GC already broke the association with the object.
     set_owner_from(NULL, DEFLATER_MARKER);
     assert(contentions() >= 0, "must be non-negative: contentions=%d", contentions());
-    _contentions = -max_jint;
+    _contentions = INT32_MIN; // minimum negative int
```

Comment:
INT_MIN (unless you want to change the type to int32_t)

Reply:
Fixed.